src/components/VParallax/VParallax.ts

```diff
@@ -45,28 +36,35 @@ export default {
 
   methods: {
     init () {
-      if (!this.$refs.img) return
+      const img = this.$refs.img as HTMLImageElement
 
-      if (this.$refs.img.complete) {
+      if (!img) return
+
+      if (img.complete) {
         this.translate()
         this.listeners()
       } else {
-        this.$refs.img.addEventListener('load', () => {
+        img.addEventListener('load', () => {
           this.translate()
           this.listeners()
         }, false)
       }
     },
     objHeight () {
-      return this.$refs.img.naturalHeight
+      return (this.$refs.img as HTMLImageElement).naturalHeight
     }
   },
 
-  render (h) {
+  render (h): VNode {
     const imgData = {
       staticClass: 'v-parallax__image',
-      style: this.styles,
+      style: {
+        display: 'block',
+        opacity: this.isBooted ? 1 : 0,
+        transform: `translate(-50%, ${this.parallax}px)`
+      },
```

Comment:
If this was moved here because you couldn't get computed to work, just have it return `object`